Simplified/NYPLAccountSignInViewController.m

```diff
@@ -715,14 +715,15 @@ - (void)validateCredentials
                          NSURLResponse *const response,
                          NSError *const error) {
        
+       [[UIApplication sharedApplication] setNetworkActivityIndicatorVisible:NO];
```

Comment:
I am not sure where else we are using `setNetworkActivityIndicatorVisible:` in the app, but calling that method directly is generally not a good idea. The problem is that multiple parts of the app can toggle the activity indicator at the same time and step on each other in the process.

If you want to make use of the activity indicator, I'd suggest wrapping up calls to `setNetworkActivityIndicatorVisible:` with a new method that manages a private counter. When passing `YES`, turn the indicator on and increment the counter. When passing `NO`, decrement the count and turn the indicator off if the counter is equal to 0. If `NO` is passed when the counter is already 0, leave the counter at 0 and log an error.